### ISSUE TITLE
Implement CodeClimate into the local workflow, not just wait on remote PRs

### ISSUE THREAD

**Issue by RafaPolit** [huridocs/uwazi#2379]
Waiting for the PR to give CodeClimate issues is not good enough.  Invest a little to try to install locally.  Mac environment is already covered, try to work on Linux.

**Comment by RafaPolit** [comment]
Each dev should do this as part of their local installation.
